app/ledger.js

```diff
@@ -761,8 +762,12 @@ var run = (delayTime) => {
 
   try {
     winners.forEach((winner) => {
-      var result = client.vote(winner)
+      var result
+      var siteSetting = appStore.getState().get('siteSettings').get(`https?://${winner}`)
```

Comment:
great point! fixed in https://github.com/brave/browser-laptop/commit/86583f1ec1bc6d7127cc8632c2fb727a1b39d98d